http/notification_endpoint.go

```diff
@@ -43,11 +43,15 @@ func NewNotificationEndpointBackend(b *APIBackend) *NotificationEndpointBackend
 	}
 }
 
+func (b *NotificationEndpointBackend) Log() *zap.Logger {
```

Comment:
I think this method should be called `Logger()` since you're returning a `Logger` not a `Log`.